libraries/botbuilder-dialogs/tests/componentDialog.test.js

```diff
@@ -87,7 +87,7 @@ describe('ComponentDialog', function () {
         const startDialog = new WaterfallDialog('start', [
             async (step) => {
                 simpleStepContextCheck(step);
```

Comment:
Believe the intent of this test is to ensure status of 'complete' is returned when calling endComponent(). Since we now always return 'cancelled' when the purpose of calling endComponent() is to cancel, updating this test to cover endComponent() when cancellation is not explicit.